lib/commons/matches/node-name.js

```diff
@@ -0,0 +1,17 @@
+/* global matches */
+let isXHTMLGlobal
+matches.nodeName = function matchNodeName (node, matcher, { isXHTML } = {}) {
+	node = node.actualNode || node;
```

Comment:
Why not use `matchesSelector` here?

Reply:
That works only if the matcher is a string, but sure. I'll do that.